dotfiles/neovim/lua/custom-rest-nvim.lua

```diff
@@ -1,64 +1,13 @@
-require("rest-nvim").setup({
```

Comment:
Updated not to need a `setup` call anymore, and the old copy-pasta from config examples that I thought I might configure someday includes options that aren't part of current config